### PR TITLE
SWS-204 Add a bit of style to the metric tab.

### PR DESCRIPTION
**Pull request by pilhuhn**
Apply some patternfly to make this look nicer until we get the replacement

bildschirmfoto 2018-03-02 um 13 21 07

### CODE REVIEW

File: src/pages/ServiceDetails/ServiceMetrics.tsx [hunk 1]
```diff
@@ -77,30 +77,44 @@ class ServiceMetrics extends React.Component<ServiceId, ServiceMetricsState> {
...
+                <h3 className="card-pf-title">
+                  <span className="fa fa-bar-chart"/>
+                  Output</h3>
+                <ul className="card-pf-body">
+                  <li>Request count rate: {this.scalar(this.state.requestCountOut)}</li>
+                  <li>Request size: {this.histogram(this.state.requestSizeOut)}</li>
+                  <li>Request duration: {this.histogram(this.state.requestDurationOut)}</li>
+                  <li>Response size: {this.histogram(this.state.responseSizeOut)}</li>
+                </ul>
+                {this.renderGrafanaLink(false)}
```
Comment (jotak) [inline src/pages/ServiceDetails/ServiceMetrics.tsx]: What a shame you missed that boolean while copy-pasting :-p

Comment (xeviknal) [inline src/pages/ServiceDetails/ServiceMetrics.tsx]: @pilhuhn there is an error here. It should be `{this.renderGrafanaLink(true)}` for showing output grafana link. Now it is showing two input links.

Comment (pilhuhn) [reply src/pages/ServiceDetails/ServiceMetrics.tsx]: Oooh :-(

Comment (pilhuhn) [reply src/pages/ServiceDetails/ServiceMetrics.tsx]: So that should be 'true' ?

Comment (theute) [reply src/pages/ServiceDetails/ServiceMetrics.tsx]: I missed it too in review and I checked all the property names (this.state.*) !

**Pull request merged**